bungiesearch/signals.py

```diff
@@ -17,32 +18,39 @@ def get_signal_processor():
         signal_class = BungieSignalProcessor
     return signal_class()
 
-__items_to_be_indexed__ = defaultdict(list)
```

Comment:
I did not know that. My understanding is that `_*_` means "protected and you shouldn't change this unless you inherit from that variable" and `__*__` means "don't touch me at all".